### ISSUE TITLE
Bad aria-label on nav element in banner

### ISSUE THREAD

**Issue by terrill** [uweb/uw-2014#133]
The nav element in the banner (with "Students", "Parents", "Faculty & Staff", etc.) has aria-label="role navigation", which is confusing since both "role" and "navigation" are ARIA terms. The value of aria-label should be a better reflection of the content and purpose of that menu. Maybe "Audience-based links"?

**Comment by lcaple** [comment]
Fixed with cbad461ce8149133d3e4c5d71811416f70e259a9. Thanks for pointing this out!
